doc/message.md

```diff
@@ -77,3 +77,46 @@ need it.)
 This API is intended for querying individual message receipts. If you are
 looking for display individual user read til which message. Please check out
 `UserConversation` section.
+
+# Message Types
+
+In android and iOS chat UIKit, following message types are supported.
+
+## Text message
+
+```
+- body (string)
+    text message content
+- metadata (json)
+    empty
+- attachment (Skygear Asset)
+    empty
+```
+
+## Image message
+
+```
+- body (string)
+    empty
+- metadata (json)
+    {
+        "width": {number of px},
+        "height": {number of px},
+        "thumbnail": {base64 encoded image}
```

Comment:
What is the expected thumbnail size? Any consense between UIKit? Better to write down here.

Reply:
Just check iOS is using 80px and android is 100px. I will change the android one and update the doc

Reply:
updated @rickmak thanks!!:)